pkg/deploy/generator/resources.go

```diff
@@ -1625,82 +1625,42 @@ func (g *generator) devCIPool() *arm.Resource {
-systemctl start docker
+rpm --import https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-7
+rpm --import https://packages.microsoft.com/keys/microsoft.asc
 
-# install jq
 yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
-yum -y install jq
-
-cat > /etc/profile.d/enablerh-git29.sh <<'EOF'
-#!/bin/bash
-if [ -x /usr/bin/scl_source ]; then
-  if [ -r /etc/scl/prefixes/rh-git29 ]; then
-	source scl_source enable rh-git29
-  fi
-fi
-EOF
 
-cat > /etc/profile.d/enablerh-python36.sh <<'EOF'
-#!/bin/bash
-if [ -x /usr/bin/scl_source ]; then
-  if [ -r /etc/scl/prefixes/rh-python36 ]; then
-  source scl_source enable rh-python36
-  fi
-fi
+cat >/etc/yum.repos.d/azure.repo <<'EOF'
+[azure-cli]
+name=azure-cli
+baseurl=https://packages.microsoft.com/yumrepos/azure-cli
+enabled=yes
+gpgcheck=yes
 EOF
 
-# install golang
-GO_VERSION=1.13.15
-curl -sL https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz -o /tmp/go${GO_VERSION}.linux-amd64.tar.gz
-mkdir -p /usr/local/go
-tar -C /usr/local/go -xzf /tmp/go${GO_VERSION}.linux-amd64.tar.gz --strip-components=1 go
-ln -s /usr/local/go/bin/* /usr/bin/
-
-# Install azp agent
-VSTS_AGENT=2.172.2
-curl -sL https://vstsagentpackage.azureedge.net/agent/${VSTS_AGENT}/vsts-agent-linux-x64-${VSTS_AGENT}.tar.gz -o /tmp/vsts-agent-linux-x64-${VSTS_AGENT}.tar.gz
-mkdir /home/cloud-user/agent && cd /home/cloud-user/agent
-tar zxvf /tmp/vsts-agent-linux-x64-${VSTS_AGENT}.tar.gz
-./bin/installdependencies.sh
-sudo chown cloud-user:root -R /home/cloud-user/agent
+yum --enablerepo=rhui-rhel-7-server-rhui-optional-rpms -y install azure-cli docker jq libassuan-devel gcc git gpgme-devel
 
-# configure agent
-sudo -u cloud-user ./config.sh --unattended --url https://dev.azure.com/msazure --auth pat --token "$CIAZPTOKEN" --pool "$CIPOOLNAME" --agent "ARO-RHEL-$HOSTNAME"
-./svc.sh install
+GO_VERSION=1.15.2
+curl -s https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz | tar -C /usr/local -xz
+ln -s /usr/local/go/bin/* /usr/local/bin
 
-# azure scripts do not work well with rhel
-# TODO: Fix upstream in vsts repos
-cat > /home/cloud-user/agent/.path <<'EOF'
-/opt/rh/rh-python36/root/usr/bin:/opt/rh/rh-git29/root/usr/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/home/cloud-user/.local/bin:/home/cloud-user/bin
-EOF
+VSTS_AGENT=2.175.2
+mkdir /home/cloud-user/agent
+pushd /home/cloud-user/agent
+curl -s https://vstsagentpackage.azureedge.net/agent/${VSTS_AGENT}/vsts-agent-linux-x64-${VSTS_AGENT}.tar.gz | tar -xz
+chown -R cloud-user:cloud-user .
 
-cat > /home/cloud-user/agent/.env <<'EOF'
-LANG=en_US.UTF-8
-LD_LIBRARY_PATH=/opt/rh/rh-python36/root/usr/lib64:/opt/rh/httpd24/root/usr/lib64
-PERL5LIB=/opt/rh/rh-git29/root/usr/share/perl5/vendor_perl
-HOME=/home/cloud-user
-EOF
+./bin/installdependencies.sh
+sudo -u cloud-user ./config.sh --unattended --url https://dev.azure.com/msazure --auth pat --token "$CIAZPTOKEN" --pool "$CIPOOLNAME" --agent "ARO-RHEL-$HOSTNAM
```

Comment:
```suggestion
sudo -u cloud-user ./config.sh --unattended --url https://dev.azure.com/msazure --auth pat --token "$CIAZPTOKEN" --pool "$CIPOOLNAME" --agent "ARO-RHEL-$HOSTNAME"
```

I think some characters got clipped off here.

Reply:
they did :-(